esp/esp/middleware/debugtoolbar/middleware.py

```diff
@@ -45,7 +45,8 @@ def custom_show_toolbar(request):
         # proxy caching. See Github issue #739.
         enabled = ((not request.is_ajax()) and settings.DEBUG_TOOLBAR and (
                 (settings.DEBUG and not request.session.get('debug_toolbar') == 'f') or
-                (request.session.get('debug_toolbar') == 't' and request.user.isAdmin())))
+                (request.session.get('debug_toolbar') == 't' and
+                (request.user.isAdmin() or request.user.isMorphed()))))
```

Comment:
Is this right?  I don't see a function `isMorphed` defined anywhere in the codebase.  However, there is a function `is_morphed` defined in `esp/esp/users/models/__init__.py` , which I think is the right function to call.

Reply:
Whoops, good catch! I even wrote down that function name in the issue...